api/controllers/SdtdServer/settings.js

```diff
@@ -48,14 +48,12 @@ module.exports = {
       server: server.id
     });
 
-    try {
-      // Force a recheck of the CPM version in the cache
-      await sails.helpers.sdtd.checkCpmVersion(server.id, true);
-
-    } catch (e) {
+    // Force a recheck of the CPM version in the cache
+    sails.helpers.sdtd.checkCpmVersion(server.id, true)
+      // No need to do anything
+      .then()
       // Ok to fail silently
-    }
-
+      .catch();
```

Comment:
pretty sure you can't pass in empty for then and catch we had that issue before

```suggestion
      .then(() => {}, () => {})
```

or even
```suggestion
      .then(Boolean,Boolean)
```

Does the function update server? Longer term does it make sense to be something like memupdate that runs occasionaly? Maybe even rename the function to not be check then?

Reply:
You're right ofcourse :smile: Added a test